optaweb-vehicle-routing-demo-app/pom.xml

```diff
@@ -15,7 +15,8 @@
   ~ limitations under the License.
   -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
That's the IntelliJ vs. VS Code formatting. Sorry for that, need to look into VS Code settings.

Reply:
I don't think it's IntelliJ vs. VS Code. I just like it better when each statement is on its own line.